packages/test-utils/src/mocks/tenure/v1/handlers.ts

```diff
@@ -18,11 +18,26 @@ export const getTenureV1 = (data: any = mockActiveTenureV1, code = 200) =>
 
 export const patchTenure = (data: any = {}, code = 200) =>
   rest.patch(`${config.tenureApiUrlV1}/tenures/:id`, (req, res, ctx) => {
-    return res(ctx.status(code), ctx.json(data));
+    const payload = req.body as Record<string, any>;
+    return res(ctx.status(code), ctx.json({ ...data, ...payload }));
```

Comment:
it might make more sense the other way around: ...payload, ...data

Reply:
it does not. Data is the initialising data, and the payload is what gets merged.